app/src/main/java/app/gamenative/ui/screen/library/RecommendedGameScreen.kt

```
@@ -0,0 +1,279 @@
+                    Column {
+                        Text(
+                            text = stringResource(summaryResId),
+                            style = MaterialTheme.typography.bodyMedium,
+                            fontWeight = FontWeight.SemiBold,
+                            color = MaterialTheme.colorScheme.onSurface,
+                        )
+                        if (game.reviewCount != null) {
+                            Text(
+                                text = stringResource(
+                                    R.string.recommended_review_count,
+                                    String.format("%,d", game.reviewCount),
```

detekt (1.23.8)

[warning] String.format("%,d", game.reviewCount) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)